test/appSearch.test.js

```diff
@@ -0,0 +1,112 @@
+			req['query']['userName'] = faker.name.findName()
+			req['query']['password'] = faker.internet.password()
+			await getBearerToken(req, resSpy)
+			authToken = resSpy.send.args[0][0]
+			headers['authorization'] = authToken
+			const response = await axios.get(`${serverUrl}/search/com.strava`, {headers: headers})
+			expect(response).toMatchObject({status: 200})
+			expect(response).toMatchObject({statusText: 'OK'})
+			expect(response.data).toEqual(expect.arrayContaining([
+				expect.objectContaining({title: 'Strava: Track Running, Cycling & Swimming'})
+			]))
+		})
+	})
+
+	describe('Search App', () => {
+		beforeEach(async () => {
+			req['query']['userName'] = faker.name.findName()
+			req['query']['password'] = faker.internet.password()
+			await getBearerToken(req, resSpy)
+			headers['authorization'] = resSpy.send.args[0][0]
+		})
+
+		it('Should pass finding multiple apps', async () => {
+			const response = await axios.get(`${serverUrl}/search/com.stra`, {headers: headers})
+			expect(response).toMatchObject({status: 200})
+			expect(response).toMatchObject({statusText: 'OK'})
+			expect(response.data).toEqual(expect.arrayContaining([
+				expect.objectContaining({title: 'Strava: Track Running, Cycling & Swimming'})
+			]))
+			expect(response.data).toEqual(expect.arrayContaining([
+				expect.objectContaining({title: 'Star Walk - Night Sky Map and Stargazing Guide'})
+			]))
+		})
+
+		it('Should pass with no apps found', async () => {
+			const response = await axios.get(`${serverUrl}/search/com.nonexistentappitem`, {headers: headers})
+			expect(response).toMatchObject({status: 200})
+			expect(response).toMatchObject({statusText: 'OK'})
+			expect(response.data).toEqual([])
+		})
+
+		describe('Limiting returned results', () => {
+			it('Should pass with only 2 results when count is an int', async () => {
+				let params = {count: 2}
+				const response = await axios.get(`${serverUrl}/search/com.stra`, {headers: headers, params: params})
+				expect(response).toMatchObject({status: 200})
+				expect(response).toMatchObject({statusText: 'OK'})
+				expect(response.data).toHaveLength(2)
+			})
+
+			it('Should pass with only 2 results when count is a string', async () => {
+				let params = {count: '2'}
+				const response = await axios.get(`${serverUrl}/search/com.stra`, {headers: headers, params: params})
+				expect(response).toMatchObject({status: 200})
+				expect(response).toMatchObject({statusText: 'OK'})
+				expect(response.data).toHaveLength(2)
+			})
+
+			it('Should fail due to count being > 250', async () => {
+				let params = {count: 260}
+				const response = await axios.get(`${serverUrl}/search/com.stra`, {headers: headers, params: params})
+				expect(response).toMatchObject({status: 200})
+				expect(response).toMatchObject({statusText: 'OK'})
+				expect(response.data).toBe('The number of results can\'t exceed 250')
+			})
+
+			it('Should fail when count is a string', async () => {
```

Comment:
I find it a bit misleading, because there is not error or something, so it's not actually 'failing'.  Maybe rename the description to 'should return no results when count is a string'